test/replaySubject.ts

```diff
@@ -0,0 +1,88 @@
+import * as assert from 'assert';
+import { unsubscribeEarly } from './helpers';
+
+import { pipe, makeReplaySubject, subscribe } from '../src/index';
+
+describe('makeReplaySubject', () => {
+  it('should deliver data if subscribed afterwards', () => {
+    const subject = makeReplaySubject<number>();
+
+    for (const x of [1, 2, 3, 4, 5]) {
+      subject(1, x);
+    }
+
+    let res: any[] = [];
+    pipe(
+      subject,
+      subscribe(
+        d => {
+          res.push(d);
+        },
+        e => {
+          if (e) assert.fail('should not terminate with error');
+        }
+      )
+    );
+
+    subject(1, 6);
+
+    assert.deepStrictEqual(res, [1, 2, 3, 4, 5, 6]);
+  });
+
+  it('should deliver data two second listener after buffer is deleted', () => {
```

Comment:
Tiny typo "two" => "to" but I wouldn't bother merging anyway

Reply:
Fixed